ads/google/doubleclick.js

```diff
@@ -164,6 +172,12 @@ function doubleClickWithGlade(global, data, gladeExperiment) {
   if (gladeExperiment === GladeExperiment.GLADE_EXPERIMENT) {
     jsonParameters.gladeExp = '1';
   }
+  const expIds = data['experimentId'];
+  if (expIds) {
+    jsonParameters.gladeEids = jsonParameters.gladeEids ?
+        jsonParameters.gladeEids + ',' + expIds : expIds;
```

Comment:
No dependency. If this is in an earlier cut than mine, the check for it existing will simply say false and gladeEids will just equal the expIds variable.